final-project/static/styles.css

```diff
@@ -0,0 +1,153 @@
+/*Final Project | styles | (jwq1)*/
```

Comment:
Look into Sass `.scss` for some more modular styling if you have the time. I don't know the state of the world in terms of build processes. (Webpack?) If it is now easier, I highly recommend.

Reply:
I found this, https://teamtreehouse.com/library/modular-css-with-sass/getting-modular-with-mixins-and-functions/course-overview.

Also, it seems like bootstrap comes with a bunch of Sass variables, which I might take advantage of. 

I am always always tempted to fix the CSS first, and have to hold myself back until I build all the HTML templates and the backend. Once everything is working, then I will spend a good chunk of time on styling.

Reply:
🏁 partially done (At least until the MVP work is done).